vivo-project/VIVO
# Move non-language-specific Vitro annotations to VIVO tbox firsttime d…

…irectory.

**[JIRA Issue](https://jira.lyrasis.org/browse/VIVO-1956)**: 

# What does this pull request do?
Moves non-language-specific vitro annotations from VIVO-languages into VIVO's home/resources/rdf/tbox/firsttime.

# What's new?
Adds file home/resources/rdf/tbox/firsttime/vitroAnnotations.n3 with non-language-tagged content.

# How should this be tested?
Should be tested in conjunction with https://github.com/vivo-project/VIVO-languages/pull/95 .
After applying both pull requests, there will be no functional changes.

The following SPARQL query should show that vitro:fullPropertyNameAnnot values now have en-US tags:

SELECT ?o WHERE {
  ?x vitro:fullPropertyNameAnnot ?o 
} ORDER BY ?o

All other vitro annotation properties should remain unchanged, but will be managed in different files.  You can test this by modifying a displayLimitAnnot value in VIVO/home/src/main/resources/rdf/tbox/firsttime , redeploying and restarting.  A query like the following will show your new value:

SELECT ?x ?o WHERE {
  ?x vitro:displayLimitAnnot ?o 
} ORDER BY ?x ?o

# Interested parties
@VIVO-project/vivo-committers
